test_1.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+
+# 2. to upload all the turtle files, from the root of this repo do:
+#    Using GNU parallel to expedite it a little
+#    Need to exclude data-config.ttl which is a config file for graphDB
+find "$PWD" -name "*.ttl" ! -name "data-config.ttl" \
+    | parallel --halt now,fail=1 --jobs 4 --cf --bar './upload.sh {}'
+
+# Serial way
+# We need to trick xargs to exit as soon as first command to fail.
+#    | xargs '-I{}' sh -c './upload.sh '{}' || exit 255'
+
+# 3,4. Run the query
+output=queries/simple2_query_output.csv
+/usr/bin/time curl --silent -X POST "${GRAPHDB_QUERY_URL}" --data-binary '@queries/simple2_query.sparql' -H 'Accept: text/csv' -H "Content-Type: application/sparql-query" >| "$output"
```

Comment:
I think this should be `GRAPHDB_API_URL` for oxigraph

Reply:
"yes" -- this is a variable @surchs added to have dedicated for query end-point, which might be other than API URL.  I am yet to adjust for oxigraph and blazegraph, will do now

Reply:
moreover I renamed "API_URL" into "UPLOAD_URL" since that is what it is used for ATM. We do not really talk to any API besides to UPLOAD and to QUERY. Hopefully that would make it cleaner.